apps/mull-api/src/database/seeders/database.seeder.ts

```diff
@@ -47,15 +47,29 @@ const US3_1Seeder = async (connection: Connection) => {
   }
 };
 
+const US8_1Seeder = async (connection: Connection) => {
+  await connection.query(
+    "INSERT INTO `mull-dev`.user (id, `password`, `email`, `timezone`, `registrationMethod`, `tokenVersion`, `joinDate`, `name`, `description`) VALUES ('8100', 'password', 'first@us81.test', '', 'LOCAL', '0', '2021-04-04 14:16:39', 'test1', '');"
+  );
+  await connection.query(
+    "INSERT INTO `mull-dev`.user (id, `password`, `email`, `timezone`, `registrationMethod`, `tokenVersion`, `joinDate`, `name`, `description`) VALUES ('8101', 'password', 'second@us81.test', '', 'LOCAL', '0', '2021-04-04 14:16:39', 'test2', '');"
+  );
+  await connection.query(
+    "INSERT INTO `mull-dev`.user (id, `password`, `email`, `timezone`, `registrationMethod`, `tokenVersion`, `joinDate`, `name`, `description`) VALUES ('8102', 'password', 'third@us81.test', '', 'LOCAL', '0', '2021-04-04 14:16:39', 'test2', '');"
+  );
+  await connection.query('INSERT INTO `mull-dev`.friends VALUES (8101, 1), (8102, 1)');
+};
+
 const US8_5Seeder = async (connection: Connection) => {
   await connection.query('INSERT INTO `mull-dev`.friends VALUES (1,2), (2,1), (1,3), (3,1)');
 };
 
 export default class DatabaseSeeder implements Seeder {
   public async run(factory: Factory, connection: Connection): Promise<any> {
-    await factory(Event)().createMany(10);
+    await factory(Event)().createMany(20);
     await createPosts(connection, factory);
     await US3_1Seeder(connection);
+    await US8_1Seeder(connection);
```

Comment:
I'm not sure who started this, but I don't like these names. At least change the name to the one you made to what it actually does.

Reply:
Well what it does is setup some users to be using for e2e tests. I think the name is fine. Do you have another name you can suggest?